pywatts/wrapper/statsmodels_wrapper.py

```diff
@@ -0,0 +1,164 @@
+    :param model_params: The fitted model parameters
+    :type xr.Dataset
+    """
+
+    def __init__(self, module: sm.tsa.base.tsa_model.TimeSeriesModel, name: str = None, module_kwargs=None,
+                 fit_kwargs=None, predict_kwargs=None, model_params=None):
+        if name is None:
+            name = module.__class__.__name__
+        super().__init__(name)
+        if module_kwargs is None:
+            module_kwargs = {}
+        if fit_kwargs is None:
+            fit_kwargs = {}
+        if predict_kwargs is None:
+            predict_kwargs = {}
+        self.module = module
+        self.module_kwargs = module_kwargs
+        self.fit_kwargs = fit_kwargs
+        self.predict_kwargs = predict_kwargs
+        self.model_params = model_params
+
+    def get_params(self) -> Dict[str, object]:
+        """
+        Returns the parameters of the statsmodels module.
+
+        :return: A dict containing the module keyword arguments, the fit keyword arguments, the predict keyword
+        arguments and the fitted model parameters
+        :rtype: dict
+        """
+        return {
+            "module_kwargs": self.module_kwargs,
+            "fit_kwargs": self.fit_kwargs,
+            "predict_kwargs": self.predict_kwargs,
+            "model_params": self.model_params
+        }
+
+    def set_params(self, module_kwargs=None, fit_kwargs=None, predict_kwargs=None, model_params=None):
+        """
+        Set the parameters of the statsmodels wrapper
+
+        :param module_kwargs: keyword arguments for the statsmodel module.
+        :param fit_kwargs: keyword arguments for the fit method.
+        :param predict_kwargs: keyword arguments for the predict method.
+        :param model_params: parameters of the fitted model
+        """
+        if module_kwargs:
+            self.module_kwargs = module_kwargs
+        if fit_kwargs:
+            self.fit_kwargs = fit_kwargs
+        if predict_kwargs:
+            self.predict_kwargs = predict_kwargs
+        if model_params:
+            self.model_params = model_params
+
+    def fit(self, x: xr.Dataset, y: xr.Dataset = None):
+        """
+        Fits the statsmodels module
+
+        :param x: input data
+        :param y: target data (not needed)
+        """
+        x = self._dataset_to_statsmodels_input(x)
+        self.model_params = self.module(x, **self.module_kwargs).fit(**self.fit_kwargs).params
+        self.is_fitted = True
+
+    @staticmethod
+    def _dataset_to_statsmodels_input(x):
+        if x is None:
+            return None
+        result = None
+        for data_var in x.data_vars:
+            data_array = x[data_var]
+            if result is not None:
+                result = np.concatenate([result, data_array.values.reshape(-1)], axis=1)
+            else:
+                result = data_array.values.reshape(-1)
+        return result
+
+    @staticmethod
+    def _statsmodels_output_to_dataset(x: xr.Dataset, prediction, name: str) -> xr.Dataset:
```

Comment:
Need this function updated after the pyWATTS API change (passing DataArrays instead of Datasets)?